exploration/semantic-skeletons.md

```diff
@@ -0,0 +1,180 @@
+# Semantic Skeletons Design
+
+Status: **Proposed**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@sffc</dd>
+    <dd>@aphillips</dd>
+		<dt>First proposed</dt>
+		<dd>2024-04-06</dd>
+		<dt>Pull Requests</dt>
+		<dd>#000</dd>
+	</dl>
+</details>
+
+## Objective
+
+_What is this proposal trying to achieve?_
+
+"Semantic skeletons" are a method introduced in CLDR 46 for programmatically selecting a datetime pattern for formatting. 
+There is a fixed set of acceptable semantic skeletons.
+
+Advantages of semantic skeletons over classical skeletons:
+
+- A smaller set of acceptable options focused on producing outputs that are sensical
+- Allows for a more efficient implementation
+- Allows for a more clear, ergonomic API
+- More future-proof since CLDR recently added them (??)
```

Comment:
This line of argumentation makes no sense. The new shiny thing being new and shiny does not mean that it'll last as long as tried and tested solutions.

Reply:
Removed the last argument. Clarified the earlier ones.